### ISSUE TITLE
GO:0007084 mitotic nuclear envelope reassembly fix parentage, refine definition

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#19950]
GO:0007084    mitotic nuclear envelope reassembly

This process occurs  is part of 'mitosis'  at the end of nuclear division

Therefore it should. be a descendant of
GO:0101024 | nuclear membrane organization involved in mitotic nuclear division
(which will be renamed "mitotic nuclear membrane organization"

Old Definition (GO:0007084)
The cell cycle process that results in reformation of the nuclear envelope during mitotic cell division.

New Definition (GO:0007084)
The mitotic cell cycle process involving ESCRTIII that results in reformation of the nuclear envelope after mitotic nuclear division. In organisms undergoing closed mitosis this involves resealing or 'repair' of the nuclear envelope in the nuclear bridge.

add synonyms
broad nuclear envelope resealing
broad nuclear envelope repair
(I made this broad, we have no idea what occurs in meiosis yet)

Closed mitosis requires local disassembly of the nuclear envelope.
Dey G, Culley S, Curran S, Schmidt U, Henriques R, Kukulski W, Baum B.
Nature. 2020 Sep;585(7823):119-123. doi: 10.1038/s41586-020-2648-3. Epub 2020 Aug 26.
PMID: 32848252

"These observations are also consistent with the highly conserved role for ESCRTIII proteins in nuclear envelope sealing and repair accfross metazoans and fungi.

**Comment by deygautam** [comment]
Sounds great!

**Comment by hattrill** [comment]
Something has gone a bit wonky here: 

mitotic nuclear membrane organization Never in Taxon | 33208 | Metazoa
is causing a taxon violation for mitotic nuclear envelope reassembly 

Not sure whether this is intentional as the definitions have changed to 'closed mitosis', but the rest of us need to know if we should be moving annotations.

**Comment by ValWood** [comment]
OK, that shouldn't have happened!

**Comment by ValWood** [comment]
I tried to make these terms open/closed mitosis agnostic so that they could be used for both....

So the open mitosis part is just qualifying what happens in open mitosis. There should be no taxon restrictions on the term.Although is this even done? The ticket is still open?

**Comment by hattrill** [comment]
That's good to know. I was a bit concerned that I'd missed something fundamental :-)

**Comment by ValWood** [comment]
This part is done, https://github.com/geneontology/go-ontology/issues/19949
but I don't see how that could have resulted in the unintended outcome. Sorry flies!

**Comment by hattrill** [comment]
No problem. I am sure it can all be fixed. Nicely caught by Protein2GO.

**Comment by pgarmiri** [comment]
Hi,
This is still flagged in Protein2GO and the taxon constrain still visible in QuickGO https://www.ebi.ac.uk/QuickGO/term/GO:0007084#ancestorChart.
Thanks,
Penelope

**Comment by ValWood** [comment]
The tax constraint should just be removed.  Everything else should be OK.

**Comment by pgaudet** [comment]
@balhoff I dont understand - there is no taxon constraint in the tsv file, but this does appear in Protégé. Can you please check what's going on ? 

![Uploading image.png…]()

**Comment by pgaudet** [comment]
Sorry - wrong term. 

The issue is coming from the parent term: 

GO:0101024	nuclear membrane organization involved in mitotic nuclear division	NCBITaxon:33208	Metazoa
GO:0101026	nuclear membrane biogenesis involved in mitotic nuclear division	NCBITaxon:33208	Metazoa

Looks like this was requested here: https://github.com/geneontology/go-ontology/issues/12728#issuecomment-253787795
Can you please confirm that this was not correct? 

Thanks, Pascale

**Comment by ValWood** [comment]
I see what has happened.

We used to have
nuclear membrane organization during closed mitosis

but I don't want to  have 'open' and 'closed' mitosis terms., because most processes are conserved.
We will only have specific terms which are required  *only* for open or closed mitosis.

So we merged
nuclear membrane organization during closed mitosis
into
nuclear membrane organization 

but unfortunatly I did not  remember that there was a taxon restriction on the child term.

The taxon restriction can be changed to "only in metazo" (but the branch should have that higher up anyway)
